.github/workflows/nightly.yml

```diff
@@ -5,11 +5,38 @@ on:
     - cron: '0 3 * * *' # 3 am UTC every day
 jobs:
   code-check:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
```

Comment:
Did you mean to reuse here .github/workflows/code-check.yml ?

Reply:
I don't like how github let's you reuse jobs because you need to specify specific branch/commit when reusing, and you can't just say use the same commit as the calling job. This makes it harder to test jobs on specific branches. That's why I think it's better to avoid reusing jobs when possible